tox.ini

```diff
@@ -38,17 +38,14 @@ passenv =
     dev: VIA_URL
     codecov: TOXENV CI TRAVIS TRAVIS_* CODECOV_*
 deps =
-    {tests,clean,coverage}: coverage
-    {tests,lint}: httpretty
-    {tests,lint,docstrings,checkdocstrings}: pytest
-    {tests,lint,docstrings,checkdocstrings}: factory-boy
-    codecov: codecov
-    lint: prospector
-    {format,checkformatting}: black
-    {docstrings,checkdocstrings}: sphinx
-    docstrings: sphinx-autobuild
-    {tests,lint,docstrings,checkdocstrings}: -r requirements.txt
-    dev: -r requirements-dev.in
+    {clean,coverage}: -r requirements-coverage.txt
```

Comment:
We should probably get rid of `clean` as a separate tox factor, it can reuse the `coverage` factor

Reply:
And the `coverage` factor should be merged into the `tests` factor